node-client/src/log.ts

```diff
@@ -0,0 +1,41 @@
+import querystring = require('querystring');
+import stream = require('stream');
+
+import { KubeConfig } from './config';
+import { WebSocketHandler } from './web-socket-handler';
+
+export class Log {
+    public 'handler': WebSocketHandler;
+
+    public constructor(config: KubeConfig) {
+        this.handler = new WebSocketHandler(config);
```

Comment:
Does the log endpoint even use WebSockets? I thought it was a standard hanging GET, that's what we do in Java:

https://github.com/kubernetes-client/java/blob/master/util/src/main/java/io/kubernetes/client/PodLogs.java#L65